Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -126,42 +108,31 @@ extension FeatureFormExampleView {
                     resultErrors = featureEditResults.errors
                 }
             } catch {
-                alertError = "The changes could not be applied to the database or table.\n\n\(error.localizedDescription)"
+                submissionError = .applyEdits(error)
                 return
             }
             if !resultErrors.isEmpty {
-                alertError = "Apply edits failed with ^[\(resultErrors.count) error](inflect: true)."
+                submissionError = .other("Apply edits returned ^[\(resultErrors.count) error](inflect: true).")
             }
             editedTables.removeAll { $0.tableName == table.tableName }
         }
     }
     
-    /// Identifies features, if any, at the current screen point.
-    /// - Parameter proxy: The proxy to use for identification.
-    /// - Returns: The first identified feature in a layer.
-    private func identifyFeature(with proxy: MapViewProxy) async -> ArcGISFeature? {
-        guard let identifyScreenPoint else { return nil }
-        let identifyLayerResults = try? await proxy.identifyLayers(
+    /// Opens a form for the first feature found at the point on the map.
+    /// - Parameter point: The point to run identify at on the map.
+    /// - Parameter map: The map to identify on.
+    private func makeFeatureForm(point: CGPoint, map: MapViewProxy) async {
```

Comment:
The name of the second parameter should be `mapView`. Also, the doc should be reformatted to use the `Parameters` delimiter.